src/components/ExportCSV/ExportCSV.jsx

```diff
@@ -0,0 +1,326 @@
+import React from 'react';
+import { ChakraProvider, Select, Button, Input, FormLabel } from '@chakra-ui/react';
+
+import './ExportCSV.css';
+
+function ExportCSV() {
+  const [sort, setSort] = React.useState('ascend');
+  const [boxes, setBoxes] = React.useState('all');
+  const [date, setDate] = React.useState('all');
+  const [launch, setLaunch] = React.useState('both');
+  const [zip, setZip] = React.useState('all');
+  const [boxesCustom, setBoxesCustom] = React.useState('');
+  const [dateCustom, setDateCustom] = React.useState('');
+  const [startCustom, setStartCustom] = React.useState('');
+  const [endCustom, setEndCustom] = React.useState('');
+  const [zipCustom, setZipCustom] = React.useState('');
+
+  // Function to check valid range
+  function checkRange(value) {
```

Comment:
I'm not able to add multiple ranges/numbers (e.g. "1-9, 12"). Make sure you are handling the case when the user enters multiple ranges or numbers. In order to do this, you may need to split the range by commas and check each range/number.